test/awss3.jl

```diff
@@ -195,6 +195,32 @@ function awss3_tests(config)
         @test s3_exists(bucket_name, key_name)
     end
 
+    @testset "Version is empty" begin
+        # Create the file to ensure we're only testing `version`
+        k = "version_empty.txt"
+        s3_put(config, bucket_name, k, "v1")
+        s3_put(config, bucket_name, k, "v2")
+
+        # Using an empty string as the version returns the latest version
+        @test s3_get(bucket_name, k; version="") == "v2"
+        @test s3_get_meta(bucket_name, k; version="") isa AbstractDict
+        @test s3_exists(bucket_name, k; version="")
+        @test s3_delete(bucket_name, k; version="") == UInt8[]
```

Comment:
I should add depredations for these even though they will continue to work for MinIO